recipe/meta.yaml

```
@@ -56,8 +60,9 @@ outputs:
       script: python -m pip install .[hub] --no-deps --ignore-installed -vv
     requirements:
       host:
+        - __unix
+        - python >=3.7
         - pip
-        - python
       run:
         - {{ pin_subpackage(name + "-base", exact=True) }}
```

Not sure about this one
```suggestion
        - __unix
        - {{ pin_subpackage(name + "-base", exact=True) }}
```

---

I think we only need to add this to the `base` package as the `pin_subpackage` bit effectively pulls the deps in.